config/task.config

```diff
@@ -0,0 +1,54 @@
+process {
+
+    withName: Cutadapt_1 {
+        ext.prefix = {"${meta.id}_1"}
+        container='quay.io/biocontainers/cutadapt:4.6--py39hf95cd2a_1'
+        ext.args =' -a "^TEST...TEST$" --discard-untrimmed '
```

Comment:
`TEST` is probably not the adapter sequences. Is this meant to get replaced by the user? If so it might be better to use a parameter for that.